lib/vfscore/automount.c

```diff
@@ -60,6 +60,7 @@
 #define LIBVFSCORE_FSTAB_VOLUME_ARGS_SEP			':'
 #define LIBVFSCORE_FSTAB_UKOPTS_ARGS_SEP			','
 #endif /* CONFIG_LIBVFSCORE_FSTAB */
+#define LIBVFSCORE_INITRD_OPT_EXTRACT				"extract"
```

Comment:
Maybe let's go with `UKOPT` instead of just `OPT`, to align it with the `ukopts` struct field member.

Reply:
I thought about doing this with ukopt but it did overall make less sense to me. The thing is that the initrd fs is already not real (based on ramfs) and then you move an option which is only specific to it (extract) to ukopt feels like distributing the options all over the place. The thing is that extract is only specific to initrd whereas mkmp is in general applicable to any filesystem. If we keep the generality of ukopt (which I thought is the idea), the implementation of any future ukopt is simpler and we do not bloat the code with condition handling.